src/jobs/characters.js

```diff
@@ -0,0 +1,22 @@
+const CronJob = require('cron').CronJob;
+import moment from 'moment';
+import models from '../models';
+import { Op } from 'sequelize';
+
+const Character = models.Character;
+
+async function deleteExpiredCharacters() {
+  const expirationDate = moment().subtract(30, 'days').toDate();
+  console.log(`Deleting characters older than ${expirationDate}...`);
+
+  await Character.destroy({
```

Comment:
We do not want to remove the record as we want to keep as much data as we can for future usage. Please only remove data that we obtained from the API.

There should actually be a command that does all this, as a clean up command PR was merged earlier. iirc the only problem left was to set up the cron job, though I am uncertain if it might also still have an issue with taking much too long to run in production due to there being several million records in the database.

That reminds me of another thing: we do not want to reset data we obtained under the old ToS.